statistics/handle/handle.go

```diff
@@ -76,7 +76,7 @@ type Handle struct {
 		memTracker *memory.Tracker
 	}
 
-	restrictedExec sqlexec.RestrictedSQLExecutor
```

Comment:
This change is needed to handle the DATA RACE problem.
If we use this single instance in different session, calling `ParseWithParams()`, the parser will be used by multiple goroutine and result in DATA RACE.